.github/workflows/python-build.yml

```diff
@@ -28,14 +28,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 flake8-markdown
+        pip install flake8 flake8-markdown flake8-docstrings
         pip install -r requirements.txt
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide.
+        # W503 and W504 are mutually exclusive. W504 is considered the best practice now.
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --ignore=D,W503
     - name: Lint with flake8-markdown
       run: |
         flake8-markdown "*.md"
+    - name: Lint with flake8-docstrings
+      run: |
+        # D212 and D213 conflict with other warni
```

Comment:
This line is incomplete.

Reply:
This line was left from when I was using `--docstring-convention all`, so the warning conflicted between the different standards. Now I use the default which is `--docstring-convention pep257`, so they don't conflict.